_integration-schemas/stripe/v1/foreign-keys.md

```diff
@@ -168,6 +172,8 @@ foreign-keys:
     attribute: "payout"
     all-foreign-keys:
       - table: "payouts"
+        join-on: "id" 
+      - table: "payout_transactions"
         join-on: "id"  
```

Comment:
This is actually `payout_transactions.payout_id`. The `payout_transactions.id` is the ID of the payout **transaction**, not the payout itself.